### PR TITLE
New class structure and names, and example

### PR DESCRIPTION
**Pull request by adamrobcarter**
Changes as discussed 10/09/24, also added [an example](https://github.com/Countoscope/countoscope/blob/cfe179ee5428550034fc8ef508d878610c3b5420/examples/example_xyz.py).

Please let me know any comments or feedback!

### CODE REVIEW

**Comment by adamrobcarter** [comment]
Things I'd particularly appreciate input on: 

Do you think it's okay to have that .xyz file in the examples folder?
Do you think the example looks okay?

File: examples/example_xyz.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+"""
+run this example from within the examples folder with
+python example_xyz.py
+"""
+
+# this minimal working example currently uses a shortened copy of datasets/brownian-particles/2D-closed/trajectory.xyz
+
+# set up the Countoscope
+countoscope = Countoscope(
+    trajectory_file  = 'trajectory2.xyz',
```
Comment (simongravelle) [inline examples/example_xyz.py]: should it be trajectory_file  = 'trajectory.xyz' (without the 2)

**Review by simongravelle — COMMENTED** [review]
Looks good to me.

**Comment by simongravelle** [comment]
@adamrobcarter feel free to merge. Then I suppose we can start working in parallel on some of the stuff we were planning to implement.

**Comment by simongravelle** [comment]
> Things I'd particularly appreciate input on:
> 
> Do you think it's okay to have that .xyz file in the examples folder? Do you think the example looks okay?

Its fine for now. 

In the future, we will have a documentation too, so then we can decide if we want to keep a documentation + an example folder. But for now, I think that its super important that we have it as it is, because other users need to know how to deal with the code.

File: examples/example_xyz.py (hunk 1 above)
Comment (adamrobcarter) [reply examples/example_xyz.py]: oops, thanks

**Comment by adamrobcarter** [comment]
Okay, I'll merge. First thing I'll work on will be data coming directly from numpy, then I'll do variable box/separation sizes

**Pull request merged**